models.js

```diff
@@ -72,16 +98,16 @@ debuglog = function () {
             };
             return toFixed(this.value.toString());
         };
+        this.toHashtag = this.toXPath;
```

Comment:
I'm not understanding the need for every kind of expression object to have both `toHashtag()` and `toXPath()` methods when the "default `xpathmodels` does not support the hashtag preprocessor." Like what does it mean to call `expr.toHashtag()` when using the default hashtag preprocessor? Also, it's unclear to me when I would want to use `expr.toHashtag()` vs `expr.toXPath()`.

Correct me if I'm wrong, but if I want to parse xpaths that contain hashtags then I must use a hashtag config object that knows about hashtags. In all other cases `XPathModels` input and output is ignorant of the concept of hashtags. From this construction, it seems that the concept of hashtags should be completely encapsulated within `HashtagExpr`, and no other expression types should need to know anything about hashtags.